### PR TITLE
Fix: Overflowing debugger content

### LINKED ISSUE #6133: [Bug] Datasources pane gets shifted when the data in the debugger is more than the set width
## Description

If the number of characters shown in the debugger pane are more than the default width of the debugger, it's size increases in a way that hides the datasources pane in the API pane.

### Steps to reproduce the behaviour:

Screenshot 2021-07-26 at 3 51 02 PM

### Important Details

- Version: [Cloud / Self-Hosted vx.x]
- OS: [e.g.MacOSX]
- Environment [production, release, deploy preview]

### PR DESCRIPTION
**Pull request by akash-codemonk**
## Description

Fixes #6133 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Manually

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>
No changes to code coverage between the base branch and the head branch</details>

### CODE REVIEW

File: app/client/src/components/editorComponents/Debugger/LogItem.tsx [hunk 1]
```diff
@@ -108,6 +108,11 @@ const JsonWrapper = styled.div`
     width: 12px !important;
     vertical-align: baseline !important;
   }
+  // This class is used react-json-view values
+  .string-value {
+    word-break: break-word;
+    overflow-wrap: anywhere;
```
Comment (rishabhsaxena) [inline app/client/src/components/editorComponents/Debugger/LogItem.tsx:114]: nitpick: I guess setting `overflow-wrap` should be enough?

Comment (akash-codemonk) [reply app/client/src/components/editorComponents/Debugger/LogItem.tsx:114]: but is has no effect in safari.

**Review by rishabhsaxena — APPROVED** [review]

**Pull request closed without merging**